src/RECONR/ReferenceGrid/src/fill.hpp

```diff
@@ -0,0 +1,14 @@
+static void fill( const double lowerBound,
+                  const double upperBound,
+                  std::vector<double>& buffer ){
+  constexpr double thresholdRatio = 1.193776641714437;
```

Comment:
`<morgan><!-- i.e., physiscist -->`That last digit, `7`, should be a `4`.`</morgan>`
`<kent><!-- i.e., engineer -->`We don't care about that much accuracy.`</kent>`
`<jeremy>`Put a comment at the end saying where that number came from.`</jeremy>`

Reply:
\<grey beard professor\> obviously the thirteenth root of 10 \</grey beard professor\>

comment incoming